applications/clouddemo/README.md

```diff
@@ -0,0 +1,11 @@
+# ASP.NET on Google Cloud
+
+This reporistory contains sample ASP.NET applications:
+
+* [`net4/`](net4/) contains a ASP.NET MVC 5 application that can be run in IIS on Windows. 
+* [`netcore/`](netcore/) contains a ASP.NET MVC Core application that can be run on Linux.
```

Comment:
and also on Windows. Core runs everywhere, this might give the impression that Core only runs on Linux.